### DISCUSSION TITLE
Interruttori luce wifi multipunto

### DISCUSSION THREAD

**Discussion by maxhrc75** [indomus/forum#2809] [GENERALE - Discussioni generiche]
Ciao ragazzi chiedo lumi per implementare tramite qualche soluzione di domotica il progettino di cui allego lo schema.
Purtroppo chi ha progettato l'impianto elettrico ha commesso gravi errori ed ora dovrei integrare senza dover smontare tutto.

Vorrei capire se è fattibile utilizzando shelly, sonoff o qualunque cosa mi possa essere d'aiuto per comandare le luci delle varie stanze.

Ho al piano terra una lampada all'ingresso. Da qui parte una scala che porta al piano di sopra dove ho un disimpegno che porta a 4 stanze.

In questo momento ci sono solo gl interruttori in ogni stanza che comandano la plafoniera all'interno, un interruttore al piano terra che comanda la lampada dell'entrata e uno che comanda il disimpegno al piano sopra. 

Avrei necessità di comandare dal piano terra la luce al disimpegno del piano sopra e viceversa e dalle stanze dovrei comandare la luce all interno della stanza e nel disimpegno.

Spero sia chiaro allego uno schema.

Grazie a chi potrà darmi dei consigli

![schema elettrico](https://github.com/indomus/forum/assets/140615842/7fcad3e4-05f1-4a24-bd3b-252102433812)

**Comment by 50494554524F** [comment]
non ho ancora studiato modifiche dirette agli impianti... idea... potresti usare bottoni wifi/bt abbinati a lampadine wifi/bt
